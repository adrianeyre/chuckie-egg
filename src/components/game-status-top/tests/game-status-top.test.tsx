import React from 'react';
import { shallow } from 'enzyme';

import GameStatusTop from '../game-status-top';
import IGameStatusTopProps from '../interfaces/game-status-top-props';

describe('Game Status Top', () => {
	it('Should render correctly', () => {
		const defaultProps: IGameStatusTopProps = {
			score: 1000,
			hiScore: 9999,
		};

		const gameStatus = shallow(<GameStatusTop {...defaultProps} />);
		expect(gameStatus).toMatchSnapshot();
	});
});