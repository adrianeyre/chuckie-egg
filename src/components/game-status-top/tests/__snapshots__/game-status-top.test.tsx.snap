// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game Status Top Should render correctly 1`] = `ShallowWrapper {}`;
